test/utils-tests.jl

```diff
@@ -144,6 +144,38 @@ let
     @test alt  ≈ 10
 end
 
+# Test eq2hor
+# The values used for the testset are from running the code. However they have been
+# correlated with the output from eq2hor routine of IDL AstroLib, with
+# differences only in the least significant digits.
+@testset "eq2hor" begin
+    alt_o, az_o, ha_o = eq2hor(259.20005705918317, 49.674706171288655, AstroLib.J2000,
```

Comment:
With original AstroLib I get:
```
GDL> eq2hor, 259.20005705918317d0, 49.674706171288655d0, 2451545d0, alt, az, ha, /B1950
% EQ2HOR: Using latitude and longitude for Pine Bluff Observatory
GDL> print, alt, az, ha, format='(f25.17)'
     43.68924703474034743
     56.68497352338920336
    291.08411539434376891
```
Are these differences due to `true_obliquity` as in this case https://github.com/JuliaAstro/AstroLib.jl/pull/31/files/821f2bd8f3c18b6a880c1919468f44afb41ac0bf#diff-95227b96f8be96e64204404da5d0a4de?